src/routes/resources/+page.svelte

```diff
@@ -63,6 +72,37 @@
     return intersection;
   }
 
+  // Update the displayedResourceLimit based on the scroll position
+  function updateDisplayLimit() {
+    const { scrollTop, clientHeight, scrollHeight } = document.documentElement;
+    const currentPosition = scrollTop + clientHeight;
+
+    if (currentPosition >= scrollHeight - SCROLL_THRESHOLD) {
+      displayedResourceLimit += DEFAULT_DISPLAY_LIMIT;
+      console.log("updateLimit");
+    }
```

Comment:
What are the units for `SCROLL_THRESHOLD`, and why 1200? Wasn't working for me until I decreased this variable (tried 200 and it worked). Does this work properly on your device?

Reply:
SCROLL_THRESHOLD is, when displayedResourceLimit (how many List item are on screen in this time) will be increased. Units are pixels distance from the bottom of the page.
I start basic from 200, but find out 1200 was more smooth for my Brave browser, but I'm not sure now. Works any way.